.github/workflows/build.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         run: |
           cp LICENSE ./build/bin/
           cp examples/run/linenoise.cpp/LICENSE ./build/bin/LICENSE.linenoise.cpp
-          zip -r llama-${{ steps.tag.outputs.name }}-bin-macos-arm64.zip ./build/bin/*
+          zip -r llama-${{ steps.tag.outputs.name }}-bin-macos-arm64.zip ./build/bin/* ./build/lib/* ./build/include/*
```

Comment:
The `build` directory is the the cmake build directory, but I don't think there is a `cmake install` or similar run at any moment, so I am not sure that it will contain `lib` and` include` directories. Am I missing something?

Reply:
I missed that. Fixing.

Reply:
It would be possible to make `cmake` output the libraries to the `bin` directory by adding `set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)` to the top level `CMakeLists.txt`. That should make easier copying the libraries, since they all over the place in some systems.

Reply:
This does not seem to make a difference on Mac - the libs still go in `/lib`:

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 42caed48..7e41a44d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,7 @@ endif()
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
     set(LLAMA_STANDALONE ON)
```

Reply:
Does it work with `CMAKE_ARCHIVE_OUTPUT_DIRECTORY` instead? I guess it's another OS difference, in Windows shared libraries are output to the RUNTIME directory instead (which is why they end in `bin` by default).

Reply:
No - still end up in `/lib`.

Reply:
I see. I was expecting the `install` target to put them in `install-prefix/bin`. Got it.